crates/steel-core/src/primitives/ports.rs

```diff
@@ -172,9 +210,10 @@ pub fn write_line(port: &SteelPort, line: &SteelVal) -> Result<SteelVal> {
 }
 
 #[function(name = "raw-write")]
-pub fn write(port: &SteelPort, line: &SteelVal) -> Result<SteelVal> {
+pub fn write(line: &SteelVal, rest: RestArgsIter<&SteelPort>) -> Result<SteelVal> {
```

Comment:
:warning: I've changed a few signatures to make the port optional.

Reply:
I think this should be fine since this is intended as an internal API

Reply:
Would it make sense to rename them to `#%raw-write`, etc.?

Reply:
probably would make more sense to do that if you were willing to include that change here, that was it is more clear that it is supposed to be largely private